Composer/packages/client/src/pages/publish/pullFailedDialog.tsx

```diff
@@ -52,6 +52,7 @@ export const PullFailedDialog: React.FC<PulledFailedDialogProps> = (props) => {
       <DialogFooter>
         <PrimaryButton text={formatMessage('Ok')} onClick={onDismiss} />
       </DialogFooter>
+      er.test
```

Comment:
is this intended?

Reply:
No, that was a typo. Thanks for catching it.